fairlearn/metrics/_metric_frame.py

```diff
@@ -189,6 +189,29 @@ class MetricFrame:
 
         .. deprecated:: 0.7.0
             `metric` will be removed in version 0.10.0, use `metrics` instead.
+
+    Examples
+    --------
+    >>> from fairlearn.metrics import MetricFrame, selection_rate
+    >>> y_true = [0,1,1,1,1,0,0,0,1,1]
+    >>> sex = ['Female']*5 + ['Male']*5
+    >>> selection_rates = MetricFrame(metrics=selection_rate,
+    ...                       y_true=y_true,
+    ...                       y_pred=y_true,
+    ...                       sensitive_features=sex)
```

Comment:
```suggestion
    >>> selection_rates = MetricFrame(
    ...      metrics=selection_rate,
    ...      y_true=y_true,
    ...      y_pred=y_true,
    ...      sensitive_features=sex)
```

At the risk of getting into really nitpicky territory... this indentation looks off. You can adjust it yourself, too, or this would be my suggestion.